spec/factories/lists/establishment_link.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :establishment_link_successor, class: 'Lists::EstablishmentLink' do
+    link_name { 'Sample' }
+    link_type { 'Successor' }
+    establishment_id { 100000 }
```

Comment:
You can just write `establishment` here and FactoryBot will create the relationship using the factory with the same name. It means there will then be actual relationships in the database.

For `linked_establishment_id` you can use something like:

```
association :linked_establishment, factory: :establishment
```